### PR TITLE
chore: delete unused event `rendered`

### PR DESCRIPTION
**Pull request by yuda110**
### To Reviewers
- [x] Skip (`style`, `chore`, `ci` ONLY)
- [x] Not that difficult

### Type of Change
- [ ] New feature
- [ ] Bug fixes
- [ ] Feature improvement
- [ ] Refactor
- [ ] Others (performance improvement, CI/CD, etc.)

### Affects to
- [ ] Packages
  - [ ] core-lib
  - [ ] mirinae
  - [ ] etc 

- [ ] Apps
  - [ ] storybook
  - [x] web

### Checklist
- Did you check the `lint` and `type`?
- Did you document the changes?
  - Changes in `mirinae` should be reflected in `storybook`.
- Did you test the app after the package changes?

### Description
SSIA

### CODE REVIEW

File: apps/web/src/services/cost-explorer/cost-analysis/modules/CostAnalysisChart.vue [hunk 1]
```diff
@@ -54,8 +51,6 @@ const state = reactive({
     legends: [] as Legend[],
     chartData: [] as XYChartData[],
     chart: null as XYChart | PieChart | null,
-    queryRef: null as null|Vue,
-    chartRef: null as null|HTMLElement,
```
Comment (piggggggggy) [inline apps/web/src/services/cost-explorer/cost-analysis/modules/CostAnalysisChart.vue:58]: question: Why did you remove this refs?

**Review by piggggggggy — APPROVED** [review]
LGTM! I left a comment!

File: apps/web/src/services/cost-explorer/cost-analysis/modules/CostAnalysisChart.vue (hunk 1 above)
Comment (yuda110) [reply apps/web/src/services/cost-explorer/cost-analysis/modules/CostAnalysisChart.vue:58]: These were for the `rendered` event, but they have all been removed since the **PDF download was removed**.
